supervisor/core/instance_test.go

```diff
@@ -0,0 +1,103 @@
+package core
+
+import (
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const testInstPath = "../../test_instances"
+const testInstName = "test_instance.lua"
+
+// commonStartTest run an Instance and checks its status.
+func commonStartTest(t *testing.T, isSignalIgnore bool) *Instance {
```

Comment:
Maybe `startTestTemplate`?

Reply:
I choose [this](https://github.com/LeonidVas/tarantool-supervisor/pull/2#discussion_r609830626) one.